packages/discord.js/src/util/Util.js

```diff
@@ -371,6 +371,22 @@ function parseWebhookURL(url) {
   };
 }
 
+/**
+ * Transforms an API auto moderation action object to a camel-cased variant.
+ * @param {APIAutoModerationAction} autoModerationAction The action to transform
+ * @returns {AutoModerationAction}
+ * @ignore
+ */
+function _transformAPIAutoModerationAction(autoModerationAction) {
```

Comment:
This would probably fit better in Transformers.ts

Reply:
Agreed! I've moved it with e08863afaaf2a12a391193a0b8852629aa18ebdd.